### PR TITLE
Feature: Social Login

### PR DESCRIPTION
**Pull request by alavinezhad**
add description here ....

### CODE REVIEW

File: file3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (alavi1970) [inline file3.txt]: This line is good but it's better if you capitalize H.

**Review by alavi1970 — CHANGES_REQUESTED** [review]
Please make these changes and do another push.

**Comment by alavinezhad** [comment]
Hello again

**Comment by alavinezhad** [comment]
ASD

**Pull request merged**